BUILD.md

```diff
@@ -264,16 +264,16 @@ cmake --build build
 1. Use a linux distribution which uses musl C. We will use Alpine Linux but any distro with musl should work. (glibc does not support static linking)
 2. Run make passing the `STATIC_BUILD` variable: `make CMAKE_EXTRA_FLAGS="-DSTATIC_BUILD=1"`
 
-In case you are not using Alpine Linux you can use a container to do the build the binary:
+In case you are not using Alpine Linux you can use a container to do the build the binary (make sure you are inside the neovim source directory):
```

Comment:
```suggestion
In case you are not using Alpine Linux you can use a container to do the build the binary (make sure you are inside the Nvim source directory):
```